python/pyspark/sql/connect/plan.py

```diff
@@ -310,6 +310,55 @@ def _repr_html_(self) -> str:
         """
 
 
+class Sample(LogicalPlan):
+    def __init__(
+        self,
+        child: Optional["LogicalPlan"],
+        lower_bound: float,
+        upper_bound: float,
+        with_replacement: bool,
+        seed: int,
+    ) -> None:
+        super().__init__(child)
+        self.lower_bound = lower_bound
+        self.upper_bound = upper_bound
+        self.with_replacement = with_replacement
+        self.seed = seed
+
+    def plan(self, session: Optional["RemoteSparkSession"]) -> proto.Relation:
+        assert self._child is not None
+        plan = proto.Relation()
+        plan.sample.input.CopyFrom(self._child.plan(session))
+        plan.sample.lower_bound = self.lower_bound
+        plan.sample.upper_bound = self.upper_bound
+        plan.sample.with_replacement = self.with_replacement
+        plan.sample.seed = self.seed
+        return plan
+
+    def print(self, indent: int = 0) -> str:
```

Comment:
For my self education on python, why do we need `print` given that we have `_repr_html`?

Reply:
I don't know either, just follow other logicalplan here